contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -801,12 +802,19 @@ linked_server_establish_connection(char *servername, LinkedServerProcess * lspro
 						 ));
 		}
 
+		/* fetch query timeout from the servername */
+		query_timeout = get_timeout_from_server_name(servername, Anum_bbf_servers_def_query_timeout);
```

Comment:
why are we fetching query timeout here? 
We are fetching it below when isTesting is false, right?